app/facades/tourist_sights_facade.rb

```diff
@@ -0,0 +1,14 @@
+class TouristSightsFacade
+  def self.sight(country)
+    tourist_data = GeoapifyService.tourist_sight(country)
```

Comment:
What youre doing here is that youre sending the country to this Service, but you need to do two things, with a country you need to get the capital, and with that capital you need to grab the lat and long and send that to the sights api.